CONTRIBUTING.md

```diff
@@ -1,4 +1,68 @@
-
+# The Build
+
+## Values
+
+We put a huge premium on a *very, very fast, perfectly consistent* build. 
+  * A typical application code change to a "leaf" module should take no more
+    than ~30 seconds to result a build status message that includes a link to a deployed version of the code.
+  * Tests runs should be consistent, and tests should pass or fail consistently *by design*. 
+    Tests should be pretty fast and cheap in terms of resources required to run.
+  * We are not currently on bazel but will likely switch to it in the future   
+
+"Ability to iterate" - critically - depends on the ability to see proposed changes in running product form. 
+  * Every successful branch build results in a deployment of the product, under a url named after the branch. 
+  * This makes changes easy to verify, review, and critique, by others.
```

Comment:
I don’t exactly know how contributing.md works, but we should make the pull request template contain the link to the branch site, maybe the checklist is dumb but it’s super cool every pr will have something you can go click around on

Reply:
Oh that's hot!

Reply:
Ok, this works via `pull_request_template.md`, and it doesn't look like there's a way to template-substitute the branch name in there.